app/main.py

```diff
@@ -1,3 +1,110 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union["Distance", float, int]) -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other, (int, float)):
+            return Distance(self.km + other)
+        else:
+            raise TypeError(
+                "Unsupported operand type(s) for +: 'Distance' and '{}'"
+                .format(type(other).__name__)
+            )
```

Comment:
```suggestion
        elif isinstance(other, (int, float)):
            return Distance(self.km + other)
        raise TypeError(
            "Unsupported operand type(s) for +: 'Distance' and '{}'"
            .format(type(other).__name__)
        )
```
in this case, it is not necessary to write `else'

Reply:
this also does not work. Can I leave it as it is, please? 

If it works == True : 
     do not touch